foundry/foundry.py

```diff
@@ -45,13 +45,15 @@ class Foundry(FoundryMetadata):
     dlhub_client: Any
     forge_client: Any
     connect_client: Any
+    index = ""
 
     xtract_tokens: Any
 
     def __init__(
         self, no_browser=False, no_local_server=False, index="mdf-test", authorizers=None, **data
     ):
         super().__init__(**data)
+        self.index = index
```

Comment:
could you just add a quick comment that includes the options available? "mdf-test", "mdf-prod"